src/services/resource.js

```diff
@@ -210,6 +358,364 @@ module.exports = class resourceHelper {
+					)
+					finalResourceIds = [...finalResourceIds, ...sequentialResourcesIds]
+
+					// fetch all resource ids assigned to another reviewer or reviewing by another reviewer
+					resourceIdsToBeRemoved = await this.findResourcesPickedUpByAnotherReviewer(
+						user_id,
+						finalResourceIds
+					)
+				}
+
+				if (resourceTypesInParallelReview.length > 0) {
+					// fetch all parallel resource ids from org
+					const parallelResourcesIds = await this.findParallelResources(
+						organization_id,
+						resourceTypesInParallelReview
+					)
+					finalResourceIds = [...finalResourceIds, ...parallelResourcesIds]
+				}
+
+				const assignedToMe = await this.findResourcesAssignedToReviewer(uniqueOrganizationIds, user_id)
+				finalResourceIds = [...finalResourceIds, ...assignedToMe]
+
+				finalResourceIds = _.difference(
+					utils.getUniqueElements(finalResourceIds),
+					utils.getUniqueElements(resourceIdsToBeRemoved)
+				)
+			}
+
+			const resourceFilter = {
+				id: { [Op.in]: finalResourceIds },
+				organization_id: { [Op.in]: uniqueOrganizationIds },
+			}
+			if (searchText != '')
+				resourceFilter.title = {
+					[Op.iLike]: '%' + searchText + '%',
+				}
+
+			if (common.TYPE in queryParams && queryParams[common.TYPE]) {
+				resourceFilter.type = queryParams[common.TYPE].split(',')
+			}
+
+			const response = await resourceQueries.resourceList(
+				resourceFilter,
+				[
+					'id',
+					'title',
+					'type',
+					'organization_id',
+					'status',
+					'user_id',
+					'submitted_on',
+					'last_reviewed_on',
+					'created_at',
+				],
+				sort,
+				page,
+				limit
+			)
+
+			const uniqueCreatorIds = utils.getUniqueElements(
+				response.result.map((item) => {
+					return item.user_id
+				})
+			)
+
+			const userDetails = await this.fetchUserDetails(uniqueCreatorIds)
+			const orgDetails = await this.fetchOrganizationDetails(uniqueOrganizationIds)
+
+			resourcesIdsNotes = await this.fetchReviewerNotesForResources(
+				user_id,
+				finalResourceIds,
+				uniqueOrganizationIds
+			)
+
+			result.data = response.result.map((item) => {
+				let returnValue = item
+				returnValue.notes = resourcesIdsNotes[item.id] || ''
+				if (inProgressResources.includes(item.id)) {
+					returnValue.reviewer_status = common.REVIEW_STATUS_INPROGRESS
+				}
+				returnValue.creator =
+					userDetails[item.user_id] && userDetails[item.user_id].name ? userDetails[item.user_id].name : ''
+				returnValue.organization = orgDetails[item.organization_id]
+				delete item.user_id
+				delete item.organization_id
+				return returnValue
+			})
+			result.count = response.count
+			result.in_progress_count = in_progress_count
+
+			return responses.successResponse({
+				statusCode: httpStatusCode.ok,
+				message: 'RESOURCE_LISTED_SUCCESSFULLY',
+				result,
+			})
+		} catch (error) {
+			throw error
+		}
+	}
+
+	static async findSequentialResources(organization_id, roles, resourceTypesInSequentialReview = []) {
```

Comment:
@adithyadinesh0412 this variable resourceTypesInSequentialReview can we rename to resourceTypes because we already know its function for sequential